libsolidity/interface/Natspec.cpp

```diff
@@ -110,9 +127,24 @@ Json::Value Natspec::devDocumentation(ContractDefinition const& _contractDef)
 				methods[it.second->externalSignature()] = method;
 			}
 		}
+		if (auto var = dynamic_cast<VariableDeclaration const*>(&it.second->declaration()))
+		{
+			Json::Value method(devDocumentation(var->annotation().docTags));
```

Comment:
Should not be called `method` - shouldn't we put it into a different dictionary ("stateVariables") anyway?

Reply:
Ah sorry, the dictionary is below.

Reply:
Will it be duplicated, though?

Reply:
I thought that we want to have the getter function of the public state variable inside `methods`. Yeah, there will be entries in `methods` and in `stateVariables` referring to the same state variable, at least for the `devdoc`.